src/main/kotlin/defineViews.kt

```diff
@@ -8,7 +8,12 @@ import com.structurizr.view.ViewSet
 fun defineViews(model: Model, views: ViewSet) {
   views.createSystemLandscapeView("system-overview", "All systems").apply {
     addAllSoftwareSystems()
-    enableAutomaticLayout(AutomaticLayout.RankDirection.TopBottom, 300, 300)
+
+    // remove 'sign-on' systems to avoid noise
+    remove(HMPPSAuth.system)
+    remove(MoJSignOn.system)
+
+    setEnterpriseBoundaryVisible(false)
```

Comment:
Are we sure we want to remove the HMPPS boundary? It feels like it's most useful in the system landscape view to show what lives with us, but I'm not particularly tied to it :)

Reply:
At the moment, the boundary is around only a few systems and I find it a bit distracting.

[This branch about using `OutsideHMPPS`](https://github.com/ministryofjustice/hmpps-architecture-as-code/compare/explicit-locations?w=1) attempts to resolve this, but I wasn't sure to wait or not... What do you think?

Reply:
After giving this some thought, I removed `setEnterpriseBoundaryVisible(false)`. It's probably better to deal with that in the linked branch/soon-to-be-PR.